unittest/liegroups.cpp

```diff
@@ -544,13 +544,13 @@ BOOST_AUTO_TEST_CASE ( test_size )
   BOOST_CHECK (se2.name () == "SE(2)");
   BOOST_CHECK (se2.neutral () == neutral);
   // SE(3): neutral = [0, 0, 0, 0, 0, 0, 1]
-  SpecialEuclideanOperationTpl <3,double> se3;
+  SpecialEuclideanOperationTpl <3,double> se3_group;
   neutral.resize (7); neutral.setZero ();
```

Comment:
To emphasize the reason why the `#define se3 pinocchio` is bad: you had to modify a unit test, which can be considered as user code.

This is not what I call compatible. I think you should remove this change.

Reply:
I have the same opinion. Let's go for that then.